specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2023-04-01-preview/loadtestservice.json

```diff
@@ -2037,6 +2037,14 @@
         "optionalLoadTestConfig": {
           "$ref": "#/definitions/OptionalLoadTestConfig",
           "description": "Configuration for quick load test"
+        },
+        "regionConfiguration": {
```

Comment:
Instead of array model, dictionary model might be more intuitive, where region could be the key and the value would be the properties / split in the region

Reply:
this would require the caller to keep track of what configuration was applied before. for ex. for yaml template, we need the entire config to be overriden